src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1047,6 +1047,17 @@ namespace cryptonote
     };
   };
 
+  struct txpool_histo
+  {
+    uint32_t txs;
+    uint64_t bytes;
+
+    BEGIN_KV_SERIALIZE_MAP()
+      KV_SERIALIZE(txs)
+      KV_SERIALIZE(bytes)
+    END_KV_SERIALIZE_MAP()
+  };
+
```

Comment:
Well, the version is meant to distinguish changes in RPC behaviour, so you bump major or minor for each change, otherwise you lose the point of it.